app/src/js/actions/index.js

```diff
@@ -582,15 +586,16 @@ export const clearPdrsSearch = () => ({ type: types.CLEAR_PDRS_SEARCH });
 export const filterPdrs = (param) => ({ type: types.FILTER_PDRS, param: param });
 export const clearPdrsFilter = (paramKey) => ({ type: types.CLEAR_PDRS_FILTER, paramKey: paramKey });
 
-export const listProviders = (options) => {
+export const listProviders = (options = {}) => {
+  const { listAll = false, queryOptions } = options;
```

Comment:
I noticed that this is _not_ using the rest params notation, but `app/src/js/actions/index.js` _is_ using rest params notation. In other words, compare this line from the other file to the line above:
```javascript
const { listAll = false, getMMT = true, ...queryOptions } = options;
```
However, both _use_ the `queryOptions` parameter afterwards in the same way. I suspect that one of these is erroneous.

Reply:
Yep, I was moving some stuff around and this got messed up in the shuffle. Nice catch!